ResearchKit/Common/ORKFormStepViewController.m

```diff
@@ -613,6 +613,18 @@ - (BOOL)allAnsweredFormItemsAreValid {
     return YES;
 }
 
+- (BOOL)isFormItemValidFor:(NSString*)formItemIdentifier {
```

Comment:
This is the only added functionality.  I created it specifically so I could check if a field's error message was no longer applicable.